### PR TITLE
e2e for consuming envs from dependency

### PR DESCRIPTION
**Pull request by AdrianPedriza**
# Proposed changes
- add e2e for -> https://github.com/okteto/okteto/pull/3593

e2e test requires a new installer version so it will not pass until https://github.com/okteto/okteto/pull/3593 generates a new installer version 

**ignore any changes that are not related to the e2e test. The other changes are reviewed in: https://github.com/okteto/okteto/pull/3593**

### CODE REVIEW

File: integration/deploy/pipeline_test.go [hunk 1]
```diff
@@ -154,6 +165,38 @@ func TestDeployPipelineManifestInsidePipeline(t *testing.T) {
 	require.True(t, k8sErrors.IsNotFound(err))
 }
 
+// TestDeployPipelineAndConsumerEnvsFromDependency tests the following scenario:
+// - Deploying a dependency
+// - Consume dependency's variable from main pipeline
+func TestDeployPipelineAndConsumerEnvsFromDependency(t *testing.T) {
```
Comment (AdrianPedriza) [inline integration/deploy/pipeline_test.go:171]: e2e using envs in main in two different ways:
- using envs from dependency `--var` variable
- using envs from dependency `OKTETO_ENV`

Comment (AdrianPedriza) [reply integration/deploy/pipeline_test.go:171]: suggestion -> https://github.com/okteto/okteto/pull/3593#discussion_r1185806556

**Review by pchico83 — APPROVED** [review]

File: integration/deploy/pipeline_test.go (hunk 1 above)
Comment (AdrianPedriza) [reply integration/deploy/pipeline_test.go]: suggestion done 9d45dc7e2c0a589e4648e41982a19cc143347704

**Comment by jLopezbarb** [comment]
I think this one is not needed anymore and can be closed. @AdrianPedriza can you confirm?

**Comment by AdrianPedriza** [comment]
> I think this one is not needed anymore and can be closed. @AdrianPedriza can you confirm?

yes it is needed. What is not needed anymore is the removal of escaped values

**Pull request closed without merging**